### PR TITLE
Changed health values so health regenerates slower.

### PR DESCRIPTION
**Pull request by Codrin-Popa**

### CODE REVIEW

File: src/server/src/system/SystemAttack.cpp [hunk 1]
```diff
@@ -222,7 +222,7 @@ void SubSystemAttack::Attack(anax::Entity attacker)
 			if (damage != 0) {
 				std::cout << "hit entity " << entity.getId().index << " for " << damage
 				          << std::endl;
-				health.HealthUpdate(-1 * damage);
+				health.HealthUpdate(-10 * damage);
```
Comment (RcColes) [inline src/server/src/system/SystemAttack.cpp]: This shouldn't be done. If you want to do more damage then change the weapons. This is just going to cause confusion later

Comment (RcColes) [reply src/server/src/system/SystemAttack.cpp]: It also breaks the printing, because it's just a terrible hack

**Comment by Codrin-Popa** [comment]
Anthony, can you try make clean and then build again?
This problem was fixed in master last week.

**Comment by AnthonyWharton** [comment]
Yeah that's what I was thinking.. I already did but I will try once again as I've been having some funky build's lately for some reason...

**Comment by AnthonyWharton** [comment]
All good, builds as expected - not sure what happened earlier. Going to address @RcColes's concerns now...

**Pull request merged**